src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,77 @@
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+  font-weight-semi-bold:          (font-weight var(--pf-global--FontWeight--semi-bold)),
+  font-weight-overpass-semi-bold: (font-weight var(--pf-global--FontWeight--overpass--semi-bold)),
+  font-weight-overpass-bold:      (font-weight var(--pf-global--FontSize--2xl))
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+
+// Font color options
+$pf-u-font-color-options: (
+  font-color-100: (color var(--pf-global--Color--100)),
+  font-color-200: (color var(--pf-global--Color--200)),
+  font-color-300: (color var(--pf-global--Color--300)),
+  font-color-400: (color var(--pf-global--Color--400)),
+);
+
+@include pf-utility-builder($pf-u-font-color-options, $pf-global--breakpoint-list);
+
+
+// Background color options
+$pf-u-font-background-color-options: (
+
+  /* Background color */
+  background-color-100:  (background-color var(--pf-global--BackgroundColor--100)),
+  background-color-200:  (background-color var(--pf-global--BackgroundColor--200)),
+
+  /*light theme*/
+  background-color-light-100:  (background-color var(--pf-global--BackgroundColor--light-100)),
+  background-color-light-200:  (background-color var(--pf-global--BackgroundColor--light-200)),
+  background-color-light-300:  (background-color var(--pf-global--BackgroundColor--light-300)),
+
+  /*dark theme*/
+  background-color-dark-100:              (background-color var(--pf-global--BackgroundColor--dark-100)),
+  background-color-dark-200:              (background-color var(--pf-global--BackgroundColor--dark-200)),
+  background-color-dark-300:              (background-color var(--pf-global--BackgroundColor--dark-300)),
+  background-color-dark-400:              (background-color var(--pf-global--BackgroundColor--dark-400)),
+  background-color-dark-transparent-100:  (background-color var(--pf-global--BackgroundColor--dark-transparent-100)),
+  background-color-dark-transparent-200:  (background-color var(--pf-global--BackgroundColor--dark-transparent-200))
+);
+
+@include pf-utility-builder($pf-u-font-background-color-options, $pf-global--breakpoint-list);
+
+
+// Font layout options
+$pf-u-font-alignment-options: (
+  font-truncate: var(--pf-m-truncate),
+  font-break-word: var(--pf-m-break-word),
+  font-nowrap: var(--pf-m-nowrap),
+  font-wrap: var(--pf-m-wrap),
+  font-fit-content: var(--pf-m-fit-content)
```

Comment:
I'm not sure about `fit-content` here... since that's a value of `width/height` or a size function used to define grid tracks, I think that's more appropriate under the sizing utility. Though it doesn't work in FF yet so I don't know if we need it yet.